probe_scraper/probe_expiry_alert.py

```diff
@@ -20,21 +22,33 @@
 FROM_EMAIL = "telemetry-alerts@mozilla.com"
 DEFAULT_TO_EMAIL = "dev-telemetry-alerts@lists.mozilla.org"
 
+BUGZILLA_BUG_URL = "https://bugzilla-dev.allizom.org/rest/bug"
+BUGZILLA_BUG_URL_2 = "https://bugzilla.mozilla.org/rest/bug"  # TODO: delete
+BUGZILLA_USER_URL = "https://bugzilla-dev.allizom.org/rest/user"
+BUGZILLA_BUG_LINK_TEMPLATE = "https://bugzilla-dev.allizom.org/show_bug.cgi?id={bug_id}"
+
 BASE_URI = "https://hg.mozilla.org/mozilla-central/raw-file/tip/toolkit/components/telemetry/"
 HISTOGRAMS_FILE = "Histograms.json"
 SCALARS_FILE = "Scalars.yaml"
 EVENTS_FILE = "Events.yaml"
 
-EMAIL_BODY_FORMAT_STRING = """
-The following Firefox probes have either expired or will expire in the next major Firefox nightly release: version {next_version} [1].
+BUG_WHITEBOARD_TAG = "probe-expiry-alert"
```

Comment:
Drive by: I think we generally use square brackets around whiteboard tags which makes them more unique (e.g. `probe-expiry-alert` would also catch `probe-expiry-alerts-on-sundays`)